nemo_text_processing/inverse_text_normalization/en_in/data/whitelist.tsv

```diff
@@ -0,0 +1,12 @@
+e.g.	for example
+dr.	doctor
```

Comment:
I think some of the entries here should be removed because we don't want them to _always_ be converted. Here are examples where the conversions make the sentence look odd:
"I love berries, like strawberries and blueberries e.g."
"I am a dr."
"watch out there mr."
"me and the mrs."
"he was a st."